Cargo.toml

```diff
@@ -22,6 +22,7 @@ members = [
     "tests/qt_types_standalone/rust",
 ]
 resolver = "2"
+msrv = "1.77.2"
```

Comment:
not sure that this is used in the cargo.toml? this would be rust-version ?

Then we need to decide if we have one version across all our crates and inherit it or whether we have a minimum per crate